v22_0/index.html

```diff
@@ -28,11 +28,11 @@
 
             //Get the Camera Option from FormIt
             (async() => {
-                const isCameraSyncOnStart = await FormIt.LumionLiveSync.GetOptions();
```

Comment:
We need this code. It is an option in the menu to camera sync on start or not. I do not approve this change.